sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ece-request/ece-process-integration.scheduler.ts

```diff
@@ -21,8 +21,6 @@ export class ECEProcessIntegrationScheduler extends BaseScheduler<void> {
   /**
    * Identifies all the applications that have an eligible COE request waiting
    * for a particular institution.
-   * @params job has generatedDate Date in which the assessment for
```

Comment:
job is a parameter right?, maybe you can adjust the comment. instead for removing